src/test/resources/application-test.yml

```diff
@@ -9,6 +9,10 @@ spring:
     default-property-inclusion: NON_EMPTY
   mustache:
     check-template-location: false
+  cloud:
```

Comment:
it's contained in application.yml, is it needed to duplicate it here?

Reply:
I am not fully confident to do this but I saw there https://github.com/folio-org/folio-spring-base/pull/77/files#diff-1bad63b7b3b478da42579fc1f09e8700cf6c258922367ac29d1f64bd90b75d19R14 so I did it